server/@types/viewModels/index.d.ts

```diff
@@ -2,11 +2,11 @@ declare module 'viewModels' {
   export interface PrisonerSummary {
     prisonNumber: string
     prisonId: string
-    releaseDate: string // TODO ideally change to a Date?
+    releaseDate?: Date
```

Comment:
The date fields are optional because sometimes we don't get values for these fields from NOMIS (prisoner search API)